src/odemis/acq/stream/_sync.py

```diff
@@ -2053,6 +2054,10 @@ def _runAcquisition(self, future):
                 s = model.DataArray(pxsum, se_md)
                 se_data.append(s)
 
+                # Live update the setting stream with the new data
+                self._se_stream._onNewData(None, se_data)
+                self._tc_stream._onNewData(None, tc_data)
```

Comment:
I don't think you should update the "se" (=e-beam) data, as for now this stream is not shown, and if you pass it as-is, it'd show a single pixel.
For the `_tc_stream`, I'd suggest to pass the dataflow as well, just to make sure we don't break anything if the function needs it (in the future). You can use `self._tc_stream._dataflow`.